### DISCUSSION TITLE
The definition of the parameter 'a' in cir method

### DISCUSSION THREAD

**Discussion by ekilcioglu** [NVlabs/sionna#350] [Q&A]
Hello,

I want to ask and clarify what the parameter 'a' represents in cir method. For example, after computing the paths and use cir method, we have the parameters 'a' channel coefficients and related delays (tau). Here, from the definition of cir, it returns the baseband equivalent signal, so here the output ('a') should be a_i^b's for each multi-path. However, in the definition of 'a' in Paths class, it seems it is the passband channel coefficients. So, my question is: after the execution of "a, tau = paths.cir()", what 'a' should I get, passband channel coefficient a_i's or baseband channel coefficients a_i^b with multiplied e^{-j2 \pi f \tau_{i}}?

Thanks for your answer in advance!

**Comment by faycalaa** [comment]
Hi,

The [`paths.cir()`](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Paths.cir) method returns the baseband equivalent CIR coefficients and corresponding delays.
